src/main/java/de/rwth/idsg/ocpp/jaxb/JavaDateTimeConverter.java

```diff
@@ -0,0 +1,35 @@
+package de.rwth.idsg.ocpp.jaxb;
+
+import de.rwth.idsg.ocpp.DateTimeUtils;
+import jakarta.xml.bind.annotation.adapters.XmlAdapter;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+
+/**
+ * Java-Time and XSD represent data and time information according to ISO 8601.
+ */
+public class JavaDateTimeConverter extends XmlAdapter<String, OffsetDateTime> {
+
+    private final ZoneId fallbackZoneId;
+    private final boolean marchallToUtc;
+
+    public JavaDateTimeConverter() {
+        this(ZoneId.systemDefault(), System.getProperty("steve.ocpp.marshall-to-utc", "true").equals("true"));
```

Comment:
i dont get why this property is necessary.

[steve has a UTC setting](https://github.com/steve-community/steve/blob/master/src/main/java/de/rwth/idsg/steve/SteveConfiguration.java#L47-L48), which has [this consequence](https://github.com/steve-community/steve/blob/master/src/main/java/de/rwth/idsg/steve/Application.java#L46-L48). finally, the library actually should just respect the zone id the java app runs in and that is it.

Reply:
The property configures the zone used by the output format: utc as expected by the specification or the default java zone.

It helps when default zone is not UTC but you want to respect the specification.

As Steve configures utc by default both configuration will have the same effect.

UTC could always be used (my 1st implementation) but your test suite checks cases with not utc as default zone.

Reply:
i would find it a little bit weird that a subsystem (this lib) behaves differently than the main java system (steve). from my point of view, there should be only one timezone configuration that applies globally to the java app/process. 

for example, if someone changes the default UTC timezone to something with UTC+2 hours such that we start adding the offset of 2h, then the date/times should reflect that new reality. we should not translate this back to UTC on OCPP messaging level.

Reply:
to prevent misunderstandings: i am not talking about `ZoneId.systemDefault()`. i see its benefit, and it must be there. i just question the system property `"steve.ocpp.marshall-to-utc"` and all the extra logic that comes with it.

now... to be fair, we probably will add a timezone config for the web UIs/pages only while letting all java logic run in UTC. therefore, all processes will run in UTC and when rendering a page they will go through the user-specific time zone config.

therefore, i find the nuance/flexibility you are adding (ocpp timestamps can do UTC, while steve runs something else) a little bit unnecessary.

Reply:
Well, I initially forced the datetime display to Zulu time as recommended by the OCPP specification, but you weren’t a fan of that.
So I made it configurable via a system property (the only viable option I found), but that didn’t go over well either.

The change is open, feel free to take what works for you and discard the rest.